src/Orchard.Web/Modules/Orchard.DesignerTools/Events/FeatureEventHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Orchard.ContentManagement;
+using Orchard.DesignerTools.Models;
+using Orchard.Environment;
+using Orchard.Environment.Extensions.Models;
+
+namespace Orchard.DesignerTools.Events {
+    public class FeatureEventHandler : IFeatureEventHandler {
+        private readonly IOrchardServices _services;
+
+        public FeatureEventHandler(IOrchardServices services) {
+            _services = services;
+        }
+
+        public void Disabled(Feature feature) {
+
+        }
+
+        public void Disabling(Feature feature) {
+
+        }
+
+        public void Enabled(Feature feature) {
+            if(feature.Descriptor.Id != "Orchard.DesignerTools") {
+                return;
+            }
+
+            // reset it to active each time the module is turned back on - less confusing for users
```

Comment:
Please use proper sentences for comments, starting with a Capital word and ending with a full stop.